components/interop/web3j-service/src/main/kotlin/net/corda/interop/web3j/internal/avalanche/AvalancheDispatchorFactory.kt

```diff
@@ -0,0 +1,59 @@
+package net.corda.interop.web3j.internal.avalanche
+
+import net.corda.interop.web3j.DispatcherFactory
+import net.corda.interop.web3j.EvmDispatcher
+import net.corda.interop.web3j.internal.EthereumConnector
+import net.corda.interop.web3j.internal.besu.GetBalanceDispatcher
+
+
+object AvalancheDispatcherFactory : DispatcherFactory {
+
+    override fun balanceDispatcher(evmConnector: EthereumConnector): EvmDispatcher {
+        return GetBalanceDispatcher(evmConnector)
+    }
+
+    override fun chainIdDispatcher(evmConnector: EthereumConnector): EvmDispatcher {
+        return AvalancheChainIdDispatcher(evmConnector)
+    }
+
+    override fun callDispatcher(evmConnector: EthereumConnector): EvmDispatcher {
+        return AvalancheCallDispatcher(evmConnector)
```

Comment:
I think this can just be
```suggestion
        return CallDispatcher(evmConnector)
```